AOS-162 Modify getmapping parsing

kometen opened the pull request.

kometen committed:
Upgrade junit-version.

erkdahl committed:
Upgraded ElasticSearchTests to Junit5

kometen committed:
Modified endpoints with proper parameter-syntax.

erkdahl commented:
For at bygget skal bygge må du forresten endre filen .github/workflows/call-build.yaml
Bytt ut gamle "on" og legg inn:
```
on:
  pull_request:
    types: [opened, edited, synchronize, reopened]
```

erkdahl committed:
Fixed build only on PR in workflow.
Cleaned up dependencies. Moved versions to properties.
Updated common-api-lib dependency.

erkdahl committed:
Fixed update-image-version workflow to use correct repo.
